app/views/sign_up_mailer/sign_up_email.text.erb

```diff
@@ -0,0 +1,21 @@
+Hi test_user,
+
+Thank you for signing up to be a Droply member. We take pride in our efforts to get you the supplies you need in the quickest manner possible.
+
+We are the place to go when peoples lives are at stake. We provide easy, fast, and cost effective supplies., so you can concentrate on saving lives.
+
+There is just a little more information need to collect, before we can begin shipping you supplies. Complete your account now, so you are prepared when disaster strikes.
+
+Thanks for your support,
+
+The Droply Team
+
+
+Want to become a registered Droply supplier? Apply Now!
+Contact Us
+
+contact@drop.ly
+
+1-555-droply
+
+1510 Blake St, Denver, CO
```

Comment:
Fix concurrent spelling errors that are mentioned in the `sign_up_email.html.erb`